samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/mesh/device_composition.c

```diff
@@ -101,6 +101,31 @@ static struct bt_mesh_elem elements[];
 
 /* message handlers (Start) */
 
+#define common()							       \
```

Comment:
There are many things wrong with this:

 - It should be a helper function
 - It should have a descriptive name matching what it does
 - Even if it was a macro it should be all upper-case

Reply:
Removed that macro. So issue has solved !!